plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -77,7 +77,7 @@ local MenuDialog = FocusManager:extend{
 function MenuDialog:init()
     self.layout = {}
     if Device:hasKeys() then
-        self.key_events.Close = { { Device.input.group.Back } }
+        self.key_events.Close = { { { "Back", "Menu" } } }
```

Comment:
@Frenzie bringing your attention here as this might affect other non-touch devices...

Reply:
Yes, we would want to maintain the full Back group.

In ButtonDialog (which is used in most places) we did this:
```lua
        if Device:hasKeys() then
            local back_group = util.tableDeepCopy(Device.input.group.Back)
            if Device:hasFewKeys() then
                table.insert(back_group, "Left")
                self.key_events.Close = { { back_group } }
            else
                table.insert(back_group, "Menu")
                self.key_events.Close = { { back_group } }
            end
```

Reply:
Changed accordingly.